modules/enable-active-backup-mode.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * networking/advanced_networking/network-bonding-considerations.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="enable-active-backup-mode_{context}"]
+= Enable active-backup mode for your cluster
+
+[role="_abstract"]
+The `active-backup` mode provides fault tolerance for network connections by switching to a backup link where the primary link fails. The mode specifies the following ports for your cluster:
+
+* An active port where one physical interface sends and receives traffic at any given time.
+* A standby port where all other ports act as backup links and continously monitor their link status.
```

Comment:
Should the list intro be in a separate paragraph so that it doesn't get included in the short desc? 

```suggestion
[role="_abstract"]
The `active-backup` mode provides fault tolerance for network connections by switching to a backup link where the primary link fails. 

The mode specifies the following ports for your cluster:
* An active port where one physical interface sends and receives traffic at any given time.
* A standby port where all other ports act as backup links and continously monitor their link status.
```